code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -115,6 +115,13 @@
 	can_hold = list(/obj/item/weapon/reagent_containers/food/snacks/grown,/obj/item/seeds,/obj/item/weapon/grown)
 	burn_state = FLAMMABLE
 
+/obj/item/weapon/storage/bag/plants/can_be_inserted(obj/item/W, stop_messages = 0, mob/living/user)
+	if(istype(W, /obj/item/weapon/reagent_containers/food/snacks/grown/cherry_bomb))

```

Comment:
Given growns are the only thing that can go into plant bags (bar seeds maybe idk), Yes.
You're starting to sound like Erro and his washing machine, "15+ istypes is SURELY better than a var! right guys? guys..?"